pycbc/conversions.py

```diff
@@ -972,10 +963,18 @@ def freq_from_final_mass_spin(final_mass, final_spin, l=2, m=2, nmodes=1):
         ``[input shape x] nmodes``, where ``input shape`` is the broadcasted
         shape of the inputs.
     """
-    return get_lm_f0tau(final_mass, final_spin, l, m, nmodes)[0]
+    if n is not None:
+        nmodes = n + 1
+    out = get_lm_f0tau(final_mass, final_spin, l, m, nmodes, which='f0')
+    if n is not None or nmodes == 1:
+        if n is None:
```

Comment:
Could we set the default `n=0` (changing the if statements accordingly)? Then we would not need this additional `if` statement

Reply:
Good point... I'll change it to default to n=0, and get rid of the extra if statement.

Reply:
I simplified this. Now all functions in conversions.py just take an `n` that can be an int or an array, like the other indices. This works with the ringdown module, since the ringdown module is only calling `get_lm_f0tau_allmodes` with the modes dictionary.